driver-service/src/main/java/by/sergo/driverservice/controller/handler/RestControllerExceptionHandler.java

```diff
@@ -0,0 +1,77 @@
+package by.sergo.driverservice.controller.handler;
+
+
+import by.sergo.driverservice.service.exception.BadRequestException;
+import by.sergo.driverservice.service.exception.NotFoundException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.rmi.ServerException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+
+@Slf4j
+@RestControllerAdvice(basePackages = {"by.sergo.driverservice.controller"})
+public class RestControllerExceptionHandler {
+
+    @ExceptionHandler(NotFoundException.class)
+    public ResponseEntity<RestErrorResponse> handleNotFoundException(NotFoundException ex) {
+        return createResponse(ex, HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler(BadRequestException.class)
+    public ResponseEntity<RestErrorResponse> handleBadRequestException(BadRequestException ex) {
+        return createResponse(ex, HttpStatus.BAD_REQUEST);
+    }
+
+    @ExceptionHandler(HttpMessageNotReadableException.class)
+    public ResponseEntity<RestErrorResponse> handleHttpMessageNotReadableException(HttpMessageNotReadableException ex) {
+        List<String> messages = new ArrayList<>();
+        messages.add("Not valid structure of JSON");
+        messages.addAll(Collections.singletonList(ex.getMessage()));
```

Comment:
Здесь можно использовать метод add
```suggestion
        messages.add(ex.getMessage());
```